lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -217,6 +217,7 @@ function dosomething_signup_create($nid, $uid = NULL, $source = NULL, $timestamp
 
   $campaign = Campaign::get($nid);
   $run = dosomething_helpers_get_current_campaign_run_for_user($nid, $user, $campaign);
+  $run->nid = 0;
```

Comment:
leftover from testing (quick hack to make it not assign run nid upon signup), thanks for spotting that